Improve archlinux support

botanegg opened the pull request:
Closes #29

Introduced 'pacman-names' field for external-deps yaml files to define ArchLinux packages with dependent libs
Filled 'pacman-names' field in external-deps yaml files for next libs:

botanegg committed:
Added ArchLinux pacman helper to external_deps plugins #29

Introduced 'pacman-names' field for external-deps yaml files to define ArchLinux packages with dependent libs

botanegg committed:
Added ArchLinux packages for external-deps #29

Filled 'pacman-names' field in external-deps yaml files for next libs:
Brotli
CryptoPP
Grpc
GssApi
Hiredis
Http_parser
Iconv
Jemalloc
LibEv
Nghttp2
OpenSSL
PostgreSQL
Protobuf
UserverGBench
UserverGTest
UserverGrpc
UserverProtobuf
ZLIB
boost
bson
c-ares
cctz
fmt
libyamlcpp
mongoc
python-dev

botanegg commented:
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/

apolukhin commented:
Looks good!

Please also update the installation instruction for Arch, it looks like `-DUSERVER_CHECK_PACKAGE_VERSIONS=0` is not required any more https://github.com/userver-framework/userver/blob/4b87e9b9875bf8d758a4997ef64120c27b2860db/scripts/docs/en/userver/tutorial/build.md?plain=1#L215

botanegg committed:
Updated installation instruction for ArchLinux

Removed -DUSERVER_CHECK_PACKAGE_VERSIONS=0 flag due to it is not required any more

apolukhin commented:
Many thanks for the wonderful work!
